src/Dialog/Dialog.js

```diff
@@ -349,7 +349,7 @@ class DialogInline extends Component {
           show={open}
           className={overlayClassName}
           style={styles.overlay}
-          onTouchTap={this.handleTouchTapOverlay}
+          onMouseDown={this.handleMouseDownOverlay}
```

Comment:
Yes, we need a different solution. We can't do that change for all the Dialog instances.